### PR TITLE
Expose fts5 rowid

### PR DESCRIPTION
**Pull request by FaFre**
Expose `rowid` of fts5 table when created with external content table (https://www.sqlite.org/fts5.html#external_content_tables).
We need the `rowid` for every insert here, and probably also for other queries.

### CODE REVIEW

File: sqlparser/lib/src/engine/module/fts5.dart [hunk 1]
```diff
@@ -28,6 +28,8 @@ class _Fts5Module extends Module {
     return _Fts5Table(
       name: stmt.tableName,
       columns: [
+        if (stmt.argumentContent.any((arg) => arg.startsWith('content_rowid')))
+          TableColumn('rowid', const ResolvedType(type: BasicType.int)),
```
Comment (simolus3) [inline sqlparser/lib/src/engine/module/fts5.dart]: I'm not sure why the `rowid` only exists when there's a `content_rowid` option? Shouldn't it be exposed whenever we have a content table in general?

**Comment by simolus3** [comment]
I've also added some static analysis in drift to make sure it creates the content table before the fts5 table and to warn if an unknown table is referenced.

I'm still not sure when the `rowid` column should be available or not. Even for fts5 tables that don't have a content table, I can do `SELECT rowid FROM <thatTable>`. I can also use `rowid` as a column for inserts in that case. Whether that's supposed to work or not I don't know :D

File: sqlparser/lib/src/engine/module/fts5.dart (hunk 1 above)
Comment (FaFre) [reply sqlparser/lib/src/engine/module/fts5.dart]: At first I got the documentation wrong, I double checked and validated that `content_rowid` is actually optional and defaults to 'rowid'

**Comment by FaFre** [comment]
Great :D
After your last comment I read the documentation again and just pushed another commit. It catches empty `content` option, and fallsback to `rowid` when `content_rowid` is not set. Input strings can contain ticks and are also getting cleaned now.

**Review by simolus3 — APPROVED** [review]
Looks good to me now, thank you!

**Pull request merged**